spike_main/spike.cc

```diff
@@ -72,6 +72,7 @@ static void help(int exit_code = 1)
   fprintf(stderr, "  --dm-no-abstract-csr  Debug module won't support abstract to authenticate\n");
   fprintf(stderr, "  --dm-no-halt-groups   Debug module won't support halt groups\n");
   fprintf(stderr, "  --dm-no-impebreak     Debug module won't support implicit ebreak in program buffer\n");
+  fprintf(stderr, "  --blocksz=<size>     Cache block size (B) for CMO operations(powers of 2) [default 16]\n");
```

Comment:
default 16 or 64?

Reply:
Agreed, @ingallsj please make a PR to fix.

Reply:
https://github.com/riscv-software-src/riscv-isa-sim/pull/923